crates/db-boot/src/x86_64/bootparam.rs

```diff
@@ -0,0 +1,3876 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the THIRD-PARTY file.
+
+/*
+ * automatically generated by rust-bindgen
```

Comment:
There are clippy warnings, so seems we need to regenerate the file with latest bindgen.

Reply:
sure, let me regenerate it.

Reply:
Done

Reply:
seems like latest version of bindgen still has this kind of clippy problem in the `test` area. I'll manually change it.

Reply:
changed.